docs/src/Callout.doc.js

```diff
@@ -22,168 +47,354 @@ card(
+          When specified, adds a dismiss button to the Callout. See the [Dismissible variant](#Dismissible) for more info.
+
+          The \`accessibilityLabel\` should follow the [Accessibility guidelines](#Accessibility).`,
       },
       {
         name: 'iconAccessibilityLabel',
         type: 'string',
         required: true,
         defaultValue: null,
-        description: [
-          'Supply a short, descriptive label for screen-readers to provide sufficient context about the callout intent.',
-          'The callout component displays `error`, `info`, or `warning` icons which visually indicate the intent of the component. `iconAccessibilityLabel` allows screen readers to communicate that intent.',
-          'Accessibility: It populates aria-label.',
-        ],
-        href: '',
+        description: `
+          Label to describe the icon’s purpose
+
+          See the [Accessibility guidelines](#Accessibility) for details on proper usage
+        `,
       },
       {
         name: 'primaryAction',
         type:
           '{| accessibilityLabel?: string, href?: string, label: string, onClick?: ({ event: SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement | SyntheticMouseEvent<HTMLButtonElement> | SyntheticKeyboardEvent<HTMLButtonElement> }) => void |}, onNavigationOptions: ({ [string]: Node | ({| +event: SyntheticEvent<> |}) => void }) => void, rel: "none" | "nofollow", target: "null" | "self" | "blank" |}',
-        required: false,
         defaultValue: null,
-        description: [
-          'Button to render inside the callout as the main call-to-action to the user. The label text has a fixed size.',
-          '- href: If href is supplied, the action will serve as a link, while if no href is supplied, the action will be a button',
-          '- label: Text to render inside the button to convey the function and purpose of the button. The button text has a fixed size.',
-          '- accessibilityLabel: Supply a short, descriptive label for screen-readers to replace button texts that do not provide sufficient context about the button component behavior. Texts like `Click Here,` `Follow,` or `Read More` can be confusing when a screen reader reads them out of context. In those cases, we must pass an alternative text to replace the button text.',
-          '- onClick: Callback fired when the button component is clicked (pressed and released) with a mouse or keyboard.',
-          `- onNavigationOptions: onNavigationOptions works in conjunction with a Provider. Pass custom props to onNavigation. See Provider for examples. onNavigation's type is flexible. Each key's value is a React.Node or an event handler function. Optional with href.`,
-          'Accessibility: `accessibilityLabel` populates aria-label. Screen readers read the `accessibilityLabel` prop, if present, instead of the button `text`.',
-        ],
+        description: `
+          Main action for people to take on the Callout
```

Comment:
ditto here on funky layout, and this first sentence needs a period (and no article)

Reply:
Main action for _users_? to take on the Callout

Reply:
Number 11 [here](https://paper.dropbox.com/doc/How-to-Convert-docs-to-new-design--BFWjPEnYkZkNBUU7ZimX79JBAg-6jXGUn10D7Ug258gdUmkp#:uid=315708423677403158321907&h2=%E2%86%92-Be-sure-to-refer-to-Pinteres) suggests it should be people. @anniemochi what do you think?